scripts/just.config.ts

```diff
@@ -13,15 +13,15 @@ import * as glob from "fast-glob";
 
 export const types = () => {
   return tscTask({
-    outDir: "lib",
+    outDir: argv().outDir || "lib",
     emitDeclarationOnly: true,
   });
 };
 
 export const build = () => {
   const baseEsbuildOptions: EsbuildBuildOptions = {
     entryPoints: glob.sync(["src/**/*.{ts,tsx}", "!src/**/__tests__/**"]),
-    outdir: "lib",
+    outdir: argv().outDir || "lib",
```

Comment:
Let's not do it this way, consistency in build systems is important. Instead, the pattern typically is that you add a commonjs file that acts as a trampoline.

i.e. add a `index.js` file in the root of the package which does something like the following:

```js
module.exports = require("./lib/index");
```

…and a `index.mjs` file which does:

```js
export * from "./lib/index.mjs"
```

Reply:
Actually, isn't this what the `main` and `module` keys are for in the `package.json` file?